.github/workflows/test.yml

```diff
@@ -17,10 +17,32 @@ jobs:
         integration: 'nri-apache'
         upgrade: false
   windows:
-    runs-on: ubuntu-latest
+    runs-on: windows-2019
     name: A job to test created packages
+    strategy:
+      fail-fast: false
+      matrix:
+        goarch: [amd64,386]
+        # We need a clean runner (not using containers here) to execute each installation test type.
+        test-upgrade: [true,false]
+        exclude:
+          #386 packages are not available for integrations. This should be removed after the first package is uploaded.
+          - goarch: 386
+            test-upgrade: true  
     steps:
       - uses: actions/checkout@v2
+      - name: Simulating Build
+        shell: pwsh
+        run: |
+          New-Item -Name "test" -ItemType "directory"
+          [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+          Invoke-WebRequest "https://github.com/newrelic/nri-vsphere/releases/download/v3.0.1/nri-vsphere-${{ matrix.goarch }}.3.0.1.msi" -OutFile "test\nri-vsphere-${{ matrix.goarch }}.3.0.1.msi"
+          ls .\test\
       - uses: ./windows
         with:
-          message: 'Totally windows'
+          tag: 'v3.0.1'
+          integration: 'vsphere'
```

Comment:
Linux action requires the `nri-` prefix. We can change either but they should be consistent to avoid confusing ourselves.

Reply:
is true, all scripts in the pipeline consider the INTEGRATION var without the nri- prefix. We can change the linux one here to be consistent with the rest. wdyt?

Reply:
just realized that this code is separate from the scripts, not need to keep that consistant and will be more generic, so i will change it